code/__DEFINES/reagents.dm

```diff
@@ -184,12 +184,6 @@
 #define GIN 			"gin"
 #define ABSINTHE 			"absinthe"
 #define RUM 			"rum"
-#define TEQUILA 			"tequila"
-#define VERMOUTH 			"vermouth"
-#define WINE 			"wine"
-#define COGNAC 			"cognac"
-#define HOOCH 			"hooch"
-#define ALE 			"ale"
```

Comment:
These are duplicates because whichever moron made all the deadrum variants decided that they should use the same IDs as the original reagents, which needs a fix, actually.